official/resnet/ctl/ctl_imagenet_benchmark.py

```diff
@@ -139,6 +139,24 @@ def benchmark_8_gpu(self):
     FLAGS.datasets_num_private_threads = 14
     self._run_and_report_benchmark()
 
+  def benchmark_8_gpu_amp(self):
+    """Test Keras model with eager, dist_strat and 8 GPUs with automatic mixed
```

Comment:
I don't know why the lint test doesn't catch this, but the first sentence of a docstring should only span 1 line. The first line must end in a period.

You can shorten this sentence to, e.g., 

```
"""Test Keras model with eager, 8 GPUs and automatic mixed precision."""
```